### DISCUSSION TITLE
Can the cloud version of Outlinewiki restrict IP?

### DISCUSSION THREAD

**Discussion by Degames-Sans** [outline/outline#5464] [Self Hosting] [answered]
Once I purchase and start using cloud version, is it possible to restrict users to access Outlinewiki only from specific IP addresses? 

For example, User A should only be able to access Outlinewiki from the IP address 34.92.126.195/32. Additionally, they should provide the correct username and password to successfully log in. 

If User A attempts to access Outlinewiki from any other IP address, they should be unable to log in.

I would like to know if it is feasible to implement this restriction and whether it can be configured by our administrators or if it requires assistance from the official Outlinewiki support team.
Thanks

**Comment by tommoor** [comment, marked as answer]
Hi, neither cloud or self-hosted has IP restrictions built in – of course if you self-host you can place whatever type of additional authentication you want in front of the server.
